lib/doorkeeper/orm/active_record/stale_records_cleaner.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Doorkeeper
+  module Orm
+    module ActiveRecord
+      class StaleRecordsCleaner
+        def initialize(model)
+          @model = model
+        end
+
+        def clean_revoked
+          table = @model.arel_table
+          @model.where.not(revoked_at: nil)
+                .where(table[:revoked_at].lt(Time.current))
+                .delete_all
+        end
+
+        def clean_expired(ttl)
```

Comment:
@nbulaj first of all sorry for commenting already merged pull request. 

IMO people will use this rake task when database is already bloated but in such case it can easily kill production database because doorkeeper does not generate index for `AccessToken#created_at` field. Maybe we should consider adding this index by default or add warning in `Readme.md` ?

Reply:
@NobodysNightmare it definitely makes sense. Maybe `find_each` could help us. Would you like to produce a PR?

Reply:
Fixed in https://github.com/doorkeeper-gem/doorkeeper/pull/1234

Reply:
Sorry for not responding. I had this somewhere in my inbox with "I want to answer there" :)

Cool that you already went ahead and fixed it :+1: